host_v2/src/components/AnswerCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { useTheme } from '@mui/material/styles';
+import { Typography, Box } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import BodyCardContainerStyled from '../lib/styledcomponents/BodyCardContainerStyled';
+import BodyCardStyled from '../lib/styledcomponents/BodyCardStyled';
+import AnswerOptionStyled from '../lib/styledcomponents/AnswerOptionStyled';
+
+interface AnswerCardProps {
```

Comment:
Given that some of these values might be null in some cases, I think our interface should reflect that? Like instructions: string[] | null or whatever values are sometimes not going to be provided to the component.